common/src/main/java/com/gentics/mesh/router/ProjectsRouterImpl.java

```diff
@@ -68,7 +68,7 @@ public Router addProjectRouter(String name) throws InvalidNameException {
 			projectRouters.put(name, projectRouter);
 			log.info("Added project router {" + name + "}");
 
-			projectRouter.route().handler(ctx -> {
+			projectRouter.route().blockingHandler(ctx -> {
```

Comment:
during my load tests I found that the db could block for a long time here and block the event loop thread

Reply:
You should use the call blockingHandler(handler, boolean) and set the boolean (ordered) to "false". Otherwise, such calls might get queued, even if enough worker pool threads are available.